lib/tenstorrent/bm_event/Kconfig

```
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+config TT_BM_EVENT
+	bool "BM event support"
+	help
+	  BM event support.
```

Didn't see this on Friday, but I'll approve anyway. Feel free to add it now or in a follow-up PR.

```suggestion
config TT_BM_EVENT
	bool "BM event support"
	depends on EVENTS
	help
	  BM event support.
```

---

Fixed!